<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Panel Finder 0.2 Beta</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>791</width>
      <height>501</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
	background-color: transparent;
	border: none;
}</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::South</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Set Input And Output</string>
     </attribute>
     <widget class="QLabel" name="text1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>1. Choose input image  :</string>
      </property>
     </widget>
     <widget class="QLabel" name="lang">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>681</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Welcome to Panel finder. This application will find potential panels from images and save them.</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>731</width>
        <height>20</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;2. Set output directory path :&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-style:italic; color:#484848;&quot;&gt;*(Must have write permission.)&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="inputImageShow">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>110</y>
        <width>291</width>
        <height>211</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ForbiddenCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: solid;
border-width: 2px;
border-color:rgb(222, 222, 222);
</string>
      </property>
      <property name="text">
       <string>                               Input Image</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="pathShow">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>330</y>
        <width>691</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>731</width>
        <height>20</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>70</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-weight: bold;
font-size: large;</string>
      </property>
      <property name="text">
       <string>Initialize</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>400</y>
        <width>731</width>
        <height>20</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="imageOpen">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Open Image</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../build-Panels-Desktop-Debug/icons/gallery-symbolic.svg</normaloff>../build-Panels-Desktop-Debug/icons/gallery-symbolic.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="directoryOpen">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>250</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Select Path</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../build-Panels-Desktop-Debug/icons/add.svg</normaloff>../build-Panels-Desktop-Debug/icons/add.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>420</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>NanumGothic</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom-style: solid;
border-bottom-width: 6px;
border-bottom-color: rgb(42, 230, 0);
</string>
      </property>
      <property name="text">
       <string>Start Processing</string>
      </property>
      <property name="iconSize">
       <size>
        <width>42</width>
        <height>42</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonFollowStyle</enum>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
      <property name="arrowType">
       <enum>Qt::RightArrow</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="spell_check">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>360</y>
        <width>681</width>
        <height>22</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Spell Check Method. (Check for spelling to detect actual words and available suggestions)</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="draw_text">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>380</y>
        <width>621</width>
        <height>22</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Write Text On Image Parts. (Inside selected directory) </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Results</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>791</width>
        <height>381</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>789</width>
         <height>379</height>
        </rect>
       </property>
       <widget class="QTextEdit" name="log_view_text">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>0</y>
          <width>391</width>
          <height>371</height>
         </rect>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: transparent;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
       <widget class="QLabel" name="response">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>0</y>
          <width>281</width>
          <height>181</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="boxes">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>180</y>
          <width>281</width>
          <height>191</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="Line" name="line_4">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>10</y>
          <width>20</width>
          <height>361</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="Line" name="line_5">
        <property name="geometry">
         <rect>
          <x>27</x>
          <y>170</y>
          <width>321</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="open_mser">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>0</y>
          <width>281</width>
          <height>181</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="open_box">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>180</y>
          <width>281</width>
          <height>191</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="result1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>771</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Panels</string>
     </attribute>
     <widget class="QComboBox" name="panelList">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>301</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToMinimumContentsLength</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>150</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="review">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>131</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Open / Edit</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../build-Panels-Desktop-Debug/icons/media-playlist-repeat.svg</normaloff>../build-Panels-Desktop-Debug/icons/media-playlist-repeat.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="save">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>26</y>
        <width>131</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Edit</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../build-Panels-Desktop-Debug/icons/compose.svg</normaloff>../build-Panels-Desktop-Debug/icons/compose.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="remove">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>30</y>
        <width>131</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove Part</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../build-Panels-Desktop-Debug/icons/delete.svg</normaloff>../build-Panels-Desktop-Debug/icons/delete.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="Line" name="line_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>751</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="panelText">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>691</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Display</string>
     </attribute>
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Draw Now</string>
      </property>
     </widget>
     <widget class="QLabel" name="imageFinalDisplay">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>771</width>
        <height>411</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Alternative Draw</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonFollowStyle</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
